core/controllers/galleries.py

```diff
@@ -95,6 +95,26 @@ def get(self):
         })
         self.render_template('galleries/gallery.html')
 
+class FullGalleryPage(base.BaseHandler):
+    """The exploration gallery page."""
```

Comment:
Rephrase the comment in line 79 of this file to `"""The home page"""` and rename that function to the HomePage
(Github won't let me comment on some lines)